packages/node/package.json

```diff
@@ -29,17 +29,18 @@
     }
   },
   "dependencies": {
+    "@edge-runtime/node-utils": "2.0.3",
     "@edge-runtime/vm": "2.0.0",
     "@types/node": "14.18.33",
     "@vercel/build-utils": "6.7.0",
-    "@vercel/node-bridge": "4.0.0",
     "@vercel/static-config": "2.0.14",
+    "async-listen": "1.2.0",
     "edge-runtime": "2.0.0",
     "esbuild": "0.14.47",
     "exit-hook": "2.2.1",
-    "node-fetch": "2.6.7",
     "ts-node": "10.9.1",
-    "typescript": "4.3.4"
+    "typescript": "4.3.4",
+    "undici": "5.21.0"
```

Comment:
I think all new `dependencies` need to be `devDependencies` unless you explicitly add to `--external` in the ncc build script

Reply:
Nevermind, I see this was changed to a dependency in https://github.com/vercel/vercel/pull/7969